platform/viewer/public/config/default.js

```diff
@@ -1,72 +1,140 @@
-window.config = {
-  // default: '/'
-  routerBasename: '/',
-  extensions: [],
-  showStudyList: true,
-  filterQueryParam: false,
-  servers: {
-    dicomWeb: [
-      {
-        name: 'DCM4CHEE',
-        wadoUriRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/wado',
-        qidoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
-        wadoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
-        qidoSupportsIncludeField: true,
-        imageRendering: 'wadors',
-        thumbnailRendering: 'wadors',
-        enableStudyLazyLoad: true,
+window.config = ({ servicesManager, dependencies }) => {
+  const { SimpleDialog, merge } = dependencies;
+
+  const callInputDialog = (data, event, callback) => {
+    const { UIDialogService } = servicesManager.services;
+
+    let dialogId = UIDialogService.create({
+      centralize: true,
+      isDraggable: false,
+      content: SimpleDialog.InputDialog,
+      useLastPosition: false,
+      showOverlay: true,
+      contentProps: {
+        title: 'Enter your annotation',
+        label: 'New label',
+        measurementData: data ? { description: data.text } : {},
+        onClose: () => UIDialogService.dismiss({ id: dialogId }),
+        onSubmit: value => {
+          callback(value);
+          UIDialogService.dismiss({ id: dialogId });
+        },
+      },
+    });
+  };
```

Comment:
I think the `cornerstone` extension should use our custom Dialog by default? This _should_ be possible to do here, but we would only do it to override the dialog the `cornerstone` extension uses by default.

Reply:
Makes sense! thanks, moved.